source/extensions/filters/common/rbac/utility.h

```diff
@@ -32,6 +32,51 @@ namespace RBAC {
 struct RoleBasedAccessControlFilterStats {
   ENFORCE_RBAC_FILTER_STATS(GENERATE_COUNTER_STRUCT)
   SHADOW_RBAC_FILTER_STATS(GENERATE_COUNTER_STRUCT)
+
+  Stats::Scope& scope_;
+  Stats::StatName per_policy_stat_;
+  Stats::StatName per_policy_shadow_stat_;
+  Stats::StatNameSetPtr stat_name_set_;
+  const Stats::StatName unknown_policy_allowed_;
+  const Stats::StatName unknown_policy_denied_;
+  const Stats::StatName unknown_shadow_policy_allowed_;
+  const Stats::StatName unknown_shadow_policy_denied_;
+
+  void add_policy(const std::string& name) {
```

Comment:
Please make method names consistent with the style guide.
```suggestion
  void addPolicy(const std::string& name) {
```